tests/helpers/test_event.py

```diff
@@ -2209,7 +2209,7 @@ def not_exist_runs_error_listener(event, updates):
     hass.states.async_set("switch.not_exist", "on")
     await hass.async_block_till_done()
 
-    assert len(syntax_error_runs) == 1
+    assert len(syntax_error_runs) == 0
```

Comment:
This template was invalid, it will never render successful. We shouldn't try over and over again